### ISSUE TITLE
Omnibus F4 remapping with custom firmware

### ISSUE THREAD

**Issue by DaPi27** [iNavFlight/inav#5279]
Hello Guys,
I am want to create a custom firmware for myself, where I want to enable softserial1 on an unusal port.
Explanation WHY I want to do that:
I have an Omnibus F4V5 Board with following setup:
- Uart1 has Gps
- Uart3 is blocked by the compass
- Uart6 is connected to my s.bus receiver.
-> Now I wanted to connect an ESP8266 wifi module.

I tried using the Omnibus F4 Pro target and enabling softserial ports. In this case CH5 and CH6 should work as RX and TX port for the softserial. So I started to solder the pins. To be sure the pins are soldered correctly I always pull lightly the wire and there it is... CH6 PWM Port just detached itself from the board.

WHAT I want to do:
Just wanted to clarify I understood things correctly: I am not using the adc-channels for rssi or current sensor. As the current sensor is mapped to PC1, I think it is possible to remove this section in the target header file and use the PC1 for my softserial1. Also, the rssi channel is mapped to PA0. As the port is an A and not a C I am wondering if I can use that port also for softserial. Later I have to find another port to connect a servo but that is another story...

Kind Regards
Daniel

**Comment by DzikuVx** [comment]
You can probably use different pin as long there will be no timer clash with anything else. In this prticular case you should refer to STM32F405 datasheet and see your options.
I do, however, suggest getting a modern hardware that has everything you need without a need to tinker with custom firmwares
